bliss/encoder.py

```diff
@@ -137,6 +151,16 @@ def get_input_tensor(self, batch):
             ), "concat_psf_params specified but psf params not present in data"
             n, _, h, w = batch["images"].size()
             inputs.append(batch["psf_params"].view(n, 6, 1, 1).expand(n, 6, h, w))
+        if self.input_transform_params.get("z_score"):
+            assert (
+                batch["background"][0, 0].std() > 0
+            ), "Constant backgrounds not supported for multi-band encoding"
+            inputs[0] = (
+                batch["images"] - torch.mean(batch["images"], dim=(2, 3), keepdim=True)
+            ) / torch.std(batch["images"], dim=(2, 3), keepdim=True)
```

Comment:
Woah, is the z-scoring computing the standard deviation over the whole image? I thought we were just dividing by the square root of the background brightness at each pixel. Do this normalization scheme work better than just subtracting the background and dividing by the square root of the back intensity?

Reply:
This is the z-scoring I've been doing for all of the z-scored models. I haven't tried subtracting the background and dividing by the square root of the background intensity. Should I experiment with that first?

Reply:
Yes please. Subtracting the background and dividing by the square root of the background intensity (per pixel) seems a lot less mysterious/complex/hard-to-analyze. Let's hope it works just as well.